validation-test/stdlib/StringUTF8.swift

```diff
@@ -0,0 +1,98 @@
+// RUN: %empty-directory(%t)
+// RUN: if [ %target-runtime == "objc" ]; \
+// RUN: then \
+// RUN:   %target-clang -fobjc-arc %S/Inputs/NSSlowString/NSSlowString.m -c -o %t/NSSlowString.o && \
+// RUN:   %target-build-swift -I %S/Inputs/NSSlowString/ %t/NSSlowString.o %s -Xfrontend -disable-access-control -o %t/String; \
```

Comment:
Please don't use `-disable-access-control` for execution tests. It doesn't work with parseable interfaces (for obvious reasons).

I'd also suggest not using interesting shell scripting in new tests, since that makes the Windows port harder. You could just unconditionally compile NSSlowString.m and `#ifdef` most of it out on non-Apple platforms.

Reply:
I'll drop the `-disable-access-control`

Even better, let's sink `NSSlowString` into StdlibUnittest: https://bugs.swift.org/browse/SR-9476